### ISSUE TITLE
Hi

### ISSUE THREAD

**Issue by pentagramz** [reoky/android-crackme-challenge#10]
Hi reoky,

since the challenge three app connect to rit.edu domain are we authorized to solve the challenge since challenge involves editing the traffic?

**Comment by reoky** [comment]
Great question, the challenge is to perform MITM on the client end, and which does affect RIT, or any of their actual servers. In my case, I did this challenge inside an airgapped lab, and thus, there was no actual connectivity, or possibility of affecting my school.

An easy way to solve this is with your own network and the latest MITMProxy. (There's a brand new version. =])

**Comment by pentagramz** [comment]
yes i already solved it

**Comment by pentagramz** [comment]
Can you provide solution for challenge 4 & 5.
I tried still not solved ,Hints not much useful

**Comment by pentagramz** [comment]
do u have plans to release more challenges ..Amazing ..Good work!

**Comment by pentagramz** [comment]
Lucas any update?

**Comment by reoky** [comment]
For challenge four, the goal is to get your GPS coordinates to appear in Minneapolis. In the live builds there is a bug, but you want to make your phone look like it is at 44.XX, -93.XX. It actually rounds to the nearest int:

if ((int) location.getLatitude() == 44 && (int) location.getLongitude() == -93) {}

You can use a GPS app that lets you fake your location to make it look as though you are in Minneapolis:
https://play.google.com/store/search?q=fake%20gps&c=apps&hl=en

For challenge five, you will want to decompile the apk in order to find the secret string in the application's string resources. You will want to try tools like:
https://ibotpeaches.github.io/Apktool/
http://www.droidviews.com/tutorial-how-to-decompile-and-recomplie-apk-files/

**Comment by pentagramz** [comment]
Yes for challenge 5 decompiled.did not find any string could you please
advice.

**Comment by pentagramz** [comment]
![test](https://user-images.githubusercontent.com/7072014/36959484-508529e8-2068-11e8-8b0a-df55eebb0998.PNG)

**Comment by pentagramz** [comment]
do you mean anything in this folders???

**Comment by pentagramz** [comment]
for challenge five

**Comment by pentagramz** [comment]
![string](https://user-images.githubusercontent.com/7072014/36960064-51fa8fe0-206b-11e8-8625-ac776d5c7609.PNG)

**Comment by pentagramz** [comment]
this one?

**Comment by pentagramz** [comment]
Hi Lucas any update please?

**Comment by reoky** [comment]
Yes, you found it! Sorry for the delay, I've been working on starting a company with a friend. These challenged are really taking off, so at some point, I need to make some more of them.. Got a few cool ideas..

I believe the answer was 'eight':
<string name="eight">eight</string>

**Comment by pentagramz** [comment]
How to solve challenge three?

**Comment by pentagramz** [comment]
tried logcat

**Comment by reoky** [comment]
Try putting the device on a fake Internet where rit.edu points at your own server.  Make sure the text 'so-much-snow' appears anywhere on the page. Thanks man.

**Comment by reoky** [comment]
You could configure bind and set the DNS on your phone if you don't want to do that at the gateway level. The goal is to show that the HTTP stream can be manipulated easily. More on this when I have time to redo these challenges.
